pages/database-management/experimental-features.mdx

```diff
@@ -1,57 +1,129 @@
 ---
 title: Experimental features
-description: Experimental features are new functionalities in Memgraph that carry some risk. They are still in development and may have stability issues. In case of any challenges, get in touch with your support representative for quick help.
+description: Experimental features in Memgraph represent the forefront of our innovation, offering cutting-edge functionality as we continuously enhance our product. While these features are still evolving and may experience occasional instability, our dedicated support team is ready to assist you promptly with any challenges, ensuring a seamless experience.
 ---
 
 import { Callout } from 'nextra/components'
 import { Card, Cards } from 'nextra/components'
 
 # Experimental features
 
+Memgraph has several features in the experimental phase:
+
+- [Vector search](#vector-search)
+- [Text search](#text-search)
+
 These features are actively being developed and may change or be removed in
 future releases. Use them with caution in production environments and only if
 you thoroughly understand their implications and are prepared for potential API
 changes, data loss, service interruption, or data inconsistency. It is highly
 encouraged to thoroughly test the experimental features in your environment.
 
-<Callout>
+To use experimental features in Memgraph, you first need to [enable
+them](#enabling-experimental-features). In case you experience issues or want to
+provide feedback, please [contact us](#feedback-and-support).
+
+## Enabling experimental features
+
+To enable an experimental feature in Memgraph, set `--experimental-enabled` flag
+to the appropriate value. For example, to run Memgraph in Docker container with the vector
+search feature, run the following command:
+
+```
+docker run -p 7687:7687 -p 7444:7444 memgraph/memgraph --experimental-enabled=vector-search
+```
 
-To enable multiple experimental features in Memgraph, you need to use the
-`--experimental-enabled` flag followed by the name of the feature. Here's how
-you can enable the `text-search`
-feature at the same time:
+To configure Memgraph correctly for each experimental feature, please refer to
+the docs for each feature. For example, vector search feature requires
+configuring `--experimental-config` flag as well.
 
-`--experimental-enabled=text-search`
+To enable more than one experimental feature, list them all, divided by a comma.
+For example, to enable vector search and text search features, run Memgraph in
+Docker container with the following command:
+
+```
+docker run -p 7687:7687 -p 7444:7444 memgraph/memgraph --experimental-enabled=vector-search,text-search
+```
 
 If you already have a Memgraph instance with an experimental feature enabled,
 and you want to add another one, you will need to restart Memgraph. Ensure that
 all the necessary flags are enabled during the restart.
```

Comment:
@DavIvek is this the correct flow for Docker users? (most of our users). For them, I think it's necessary to start a new container.

Reply:
I believe it is correct. I haven't changed this part of the documentation. If this was correct before, vector search shouldn't be an exception.